src/css/components/_alert.scss

```diff
@@ -63,7 +63,7 @@
     max-height: calc((#{$master-height} - 75px) * 0.6);
 
     h4 {
-        overflow: scroll;
+        overflow: auto;
```

Comment:
With auto I cannot scroll down on an alert with too much text.

For ex when I send this alert I can tell the text continues down past the bottom of the container but I cannot read that text or scroll down to it.

```
 Alert {"alertText1":"Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibu"}
```

Reply:
I'm not seeing a scroll bar on my end, even on the latest relase/0.11.0 commit, for an alert request with that value.

I don't think that alertText* should be affected by this PR since the change relates to `h4` whereas all alert text fields are `p` elements
![Screenshot from 2021-09-14 22-50-55](https://user-images.githubusercontent.com/10703941/133362757-2ddbef83-1f8a-41ab-a766-9afe2edf56ba.png)